public/modules/custom/tyollisyyspalvelut_common/css/tyollisyyspalvelut_common.css

```diff
@@ -0,0 +1,11 @@
+.formDsiplay_sidebar_notification * {
+  max-width:100%;
```

Comment:
It's unnecessary to apply the styles for every child element. Select the elements that need styling chances. Drupal admin theme has so many classes there can be some side effects. Also the remove button should be clickable. 

You can also decrease the notification fields margins/paddings if needed.